dashboard.js

```diff
@@ -232,7 +232,7 @@ module.exports = {
 		if(messageID == -1){
 			throw "Error creating dashboard, please check that the bot has permission to post in this channel.";
 		}
-		await pgClient.query("INSERT INTO dashboard (concatkey, channel, messageid, world) VALUES ($1, $2, $3, $4)\
+		pgClient.query("INSERT INTO dashboard (concatkey, channel, messageid, world) VALUES ($1, $2, $3, $4)\
```

Comment:
Might want to have an await/some more error handling here?  In the rare event that there's an issue with the database

Reply:
I'm not really sure if that needed, the original code didn't have a `try` `catch` and shouldn't the `try` `catch` in `main.js` catch any error thrown?

Reply:
My main concern is that if there's an issue in a non-awaited async I don't think it will actually throw for main to catch.  Unlikely for there to be an issue in the db that doesn't result in the entire bot crashing before this point, so yeah can probably skip handling a failure here.